src/app/tests/suites/certification/Test_TC_S_2_4.yaml

```diff
@@ -449,7 +449,7 @@ tests:
       arguments:
           values:
               - name: "message"
-                value: "Please confirm that transition to AC2 was made over 5s"
+                value: "Please confirm that transition to AC1 was made over 5s"
```

Comment:
The Transition time should be **20s** instead of **5s**,.

Reply:
Updated